app/db.py

```diff
@@ -0,0 +1,27 @@
+"""This module wraps the mongodb interface."""
```

Comment:
i'm confused. why is this file showing up as new? should already be in main.. is your branch up to date with main?